scripts/benchmarks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+steps=50
```

Comment:
Since all these variables are only used once I'd just hardcode them into the bench command